jukebox/src/main/webapp/js/script.js

```diff
@@ -1,25 +1,112 @@
-function generateGroup() {
+/**
+This file contains all database reads/writes as well some minor webpage
+modifications. It's most likely not secure to have the database available
+on the client-side, so this set-up is temporary (although will likely remain
+as apart of the MVP)
+ */
 
-    //send to server once database schema decided
-    var groupName = document.getElementById('group_name').value;
-    window.location.href = '/create_pin.html';
+//Setting up Firebase configuration
+var firebaseConfig = {
+
+    apiKey: "AIzaSyDkEJE1taOS3R_dxakJjMB98I-3Apjuhxk",
+    authDomain: "jukebox-sps.firebaseapp.com",
+    databaseURL: "https://jukebox-sps.firebaseio.com",
+    projectId: "jukebox-sps",
+    storageBucket: "jukebox-sps.appspot.com",
+    messagingSenderId: "729977109157",
+    appId: "1:729977109157:web:0f47881639cc145866ab4c",
+    measurementId: "G-9M791HTW5T"
+}
+
+// Initialize Firebase
+firebase.initializeApp(firebaseConfig);
+var db = firebase.firestore();
+
+//This function sets up the room page, getting the room name based on PIN
+function presentRoom(isHost) {
+
+    const queryParameters = new URLSearchParams(window.location.search);
+    const roomRef = db.collection('rooms').doc(queryParameters.get('pin'))
```

Comment:
you could also pull queryParameters.get('pin') into its own variable so you don't need to retrieve it twice